packages/graphics/mesa/package.mk

```diff
@@ -13,6 +13,14 @@ PKG_LONGDESC="Mesa is a 3-D graphics library with an API."
 PKG_TOOLCHAIN="meson"
 PKG_BUILD_FLAGS="+lto"
 
+case $PROJECT in
+  Amlogic)
+    PKG_VERSION="659aa3dd6519f64379e91ca97fe184434fd7fdee" # master-19.2
```

Comment:
I'd rather not have another mesa version in use. I thought the plan was to just use the mali blobs for now anyways?

Reply:
I guess there is no fully working blob for the T820 / S912 SoC so mesa kicks in?

Reply:
Yes, there are no blobs for T820 (T860 blob can be made to work only partially) and there is no Lima/Panfrost in stable Mesa. Another approach is drop S912 support and use mali blobs.

Reply:
Unless we get lucky and a redistributable T820 blob appears; two images (AMLGX/AMLG12) are the only sensible option. Both lima/panfrost are in a fluid state with frequent bug fixes and things of interest to test/prove so it makes sense to track mesa master. NB: LE master is not a release branch .. we are allowed to track development with it!